invenio_previewer/webpack.py

```diff
@@ -66,6 +68,7 @@
                 'prism_js': './js/invenio_previewer/prismjs.js',
                 'prism_css': './scss/invenio_previewer/prismjs.scss',
                 'pdfjs_js': './js/invenio_previewer/pdfjs.js',
+                'open_pdf': './js/invenio_previewer/open_pdf.js',
```

Comment:
since you are on this, what you if you try to bump `pdfjs-dist` to `>3`? (and check if it works) :)

Reply:
sure, gonna check it out when there's a free minute :)

Reply:
Unfortunately, `invenio-cli assets build` seems to fail with `npm` 6.4.13, `node` 14.17.3 (and current packages via `invenio-cli init -c master`).

`pdfjs-dist`: `^2.0`:
```
ERROR in ./node_modules/pdfjs-dist/build/pdf.js 1413:9
Module parse failed: Unexpected character '#' (1413:9)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
| class PDFDocumentLoadingTask {
>   static #docId = 0;
|
|   constructor() {
 @ ./js/invenio_previewer/pdfjs/viewer.js 35:0-32 36:0-6 37:0-6 38:0-6
 @ ./js/invenio_previewer/pdfjs.js
```

Basically the same error with `pdfjs-dist`: `^3.0`:
```
ERROR in ./node_modules/pdfjs-dist/build/pdf.js 1172:9
Module parse failed: Unexpected character '#' (1172:9)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| }
| class PDFDocumentLoadingTask {
>   static #docId = 0;
|   #onUnsupportedFeature = null;
|   constructor() {
 @ ./js/invenio_previewer/pdfjs/viewer.js 35:0-32 36:0-6 37:0-6 38:0-6
 @ ./js/invenio_previewer/pdfjs.js
```

Reply:
Same errors come up with `npm` 8.19.3 and `node` 16.19.0.

Reply:
Thank you for the time that you have put in testing this. Given the issues encountered, I guess we can work on this in a future PR.